js/main.js

```diff
@@ -0,0 +1,61 @@
+document.addEventListener('DOMContentLoaded', function() {
+	let fizz = document.querySelector("#fizz");
+  let buzz = document.querySelector("#buzz");
+  const button = document.querySelector("#button");
+
+  button.addEventListener('click' , function(){
+    let output = document.querySelector("#output");
+    output.innerHTML = "";
+    let p = document.createElement("p");
+    p.innerText = "整数値を入力してください";
+
+    if (fizz.value.match(/[a-z]+/i) || buzz.value.match(/[a-z]+/i) ){
+      output.appendChild(p);
+      return false;
+    }
+
+    else if(fizz.value.match(/^-?[0-9]+\.[0-9]+$/) || buzz.value.match(/^-?[0-9]+\.[0-9]+$/)) {
+      output.appendChild(p);
+      return false;
+    }
+
+    else if(fizz.value == "" || buzz.value == "") {
```

Comment:
```suggestion
    else if(fizz.value === "" || buzz.value === "") {
```

より厳密に比較する方が予期せぬ不具合を生まずに済むので
特別な理由がない限り `===` を使いましょう。